extensions/algolia/fieldgroups/algolia-event.schema.json

```
@@ -0,0 +1,149 @@
+{
+  "$id": "https://ns.algolia.com/xdm/algolia/fieldgroup/event",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Algolia Event Details",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:status": "experimental",
+  "meta:license": [
+    "Copyright 2024 Algolia Inc. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "description": "Algolia gathers clicks/conversation data that is used to track click/conversion evnets.  It will also unlock Algolia AI features. This field group is a standard field group that can be used to map to these events.",
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "definitions": {
+    "algolia-event": {
```

```suggestion
    "algolia-event": {
          "type": "object",
          "properties": {
            "algoliaEvent": {
```
We need to add a namespace for algolia event fields so that they won't collide with other experience event fields. To add a namespace, please wrap the entire schema definition inside an object "algoliaEvent"